addons/mail/static/src/core/common/message.xml

```diff
@@ -73,6 +73,9 @@
                         >
                             <div class="o-mail-Message-content o-min-width-0" t-att-class="{ 'w-100': state.isEditing }">
                                 <div class="o-mail-Message-textContent position-relative d-flex" t-att-class="{ 'w-100': state.isEditing }">
+                                    <div t-if="message.message_type === 'notification' and message.body" class="o-mail-Message-body text-break mb-0 w-100" name="notification body">
+                                        <t t-out="messageSearch?.highlight(message.body) ?? message.body"/>
+                                    </div>
```

Comment:
This is copy-paste of body part of `web/message_patch.xml`. Would be nice to be in its own `t-name` and `t-call` twice. The `name` attribute could be tackle by putting it on a dummy `<t>` node